package.json

```diff
@@ -47,6 +47,7 @@
     "jwt-decode": "2.2.0",
     "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#81f57c024e137879d6c93bef62308971d0ec71b0",
     "lodash": "4.17.4",
+    "moment": "^2.19.4",
```

Comment:
We pin our dependencies to a specific version, can you remove the caret?

Reply:
Fixed, thanks!